frontend/control-center/src/pages/Status/ComponentListItem/ItemInfo.tsx

```diff
@@ -0,0 +1,115 @@
+import React, {useEffect, useState} from 'react';
+import {ReactComponent as CheckmarkIcon} from 'assets/images/icons/checkmarkFilled.svg';
+import {ReactComponent as UncheckedIcon} from 'assets/images/icons/serviceUnhealthy.svg';
+import {ReactComponent as ArrowRight} from 'assets/images/icons/arrowRight.svg';
+import {ReactComponent as ArrowDown} from 'assets/images/icons/arrowDown.svg';
+import {getChannelAvatar} from '../../../components/ChannelAvatar';
+import {capitalizeTitle} from '../../../services';
+import {Toggle} from 'components';
+import {Source} from 'model';
+import styles from './index.module.scss';
+
+type ComponentInfoProps = {
+  healthy: boolean;
+  itemName: string;
+  isComponent: boolean;
+  isExpanded: boolean;
+  setIsExpanded: React.Dispatch<React.SetStateAction<boolean>>;
+  enabled?: boolean;
+};
+
+export const ItemInfo = (props: ComponentInfoProps) => {
+  const {healthy, itemName, isComponent, isExpanded, setIsExpanded, enabled} = props;
+  const [channelSource, setChannnelSource] = useState('');
+  const [componentName, setComponentName] = useState('');
+  const [componentEnabled, setComponentEnabled] = useState(enabled);
+
+  useEffect(() => {
+    if (itemName.includes('twilio')) {
```

Comment:
We need to find a better solution for this in the future, we cannot keep adding here if we have 200 components

Reply:
Yeah we can actually use a function that takes in the name of the component as input and returns the second word capitalized for the title: eg. `sources-facebook` --> `Facebook`

I will implement it!